jbosstools/multiple/jbosstools-multiple.target

```diff
@@ -468,7 +468,13 @@
       <repository location="http://download.jboss.org/jbosstools/updates/requirements/egit/v4.0.0.201505260635-rc2/"/>
       <unit id="org.eclipse.egit.ui.importer" version="4.0.0.201505271301"/>
     </location>
-
+    
+    <!-- Eclipse Docker Tooling -->
+    <location includeAllPlatforms="false" includeMode="slicer" type="InstallableUnit" includeSource="true">
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/docker/1.0.0.201506030402/"/>
+      <unit id="org.eclipse.linuxtools.docker.feature" version="1.0.0.201506030402"/>
```

Comment:
Here, the id has to be suffixed with .feature.group ("mvn clean verify -X" warns about that id, for features, you always need to append .feature.group for p2 unit name).
unit id is org.eclipse.linuxtools.docker.feature.feature.group